### DISCUSSION TITLE
Build Validation w AzureDevOps wg path

### DISCUSSION THREAD

**Discussion by szympulka** [AzureCommunityPL/group#53] [Pomoc]
Ustawiłem sobie builda, by przy każdym PR się odpalał. Chciałem go zawęzić do konkretnego katalogu wg dokumentacji, lecz coś nie bangla. Gdzie jest błąd?

![image](https://user-images.githubusercontent.com/12598823/92331984-aac5fd80-f07a-11ea-9d0b-deca1bc9232c.png)

**Comment by kamil-mrzyglod** [comment]
A próbowałeś dać `Frontend/*` zamiast `/Frontend/*`?

**Reply by szympulka** [reply]
to samo

**Reply by kamil-mrzyglod** [reply]
Rozumiem, że używasz klasycznych pipeline'ów, nie YAML?

**Reply by szympulka** [reply]
tak
